docs_app/content/tutorial/1-subscribing.md

```diff
@@ -0,0 +1,66 @@
+# 1. Subscribing to an Observable
+
+## TLDR:
+
+- Observables don't do anything until you call [subscribe](API)
+- Calling [subscribe](API) will synchronously execute code that sets up the subscription, telling the underlying [producer](GL) to start pushing values to your code (the [consumer](GL)).
+- [subscribe](API) may be called with _one function_ that is the [next](GL) handler, *OR* an [`Observer`](API) object, which has a [next](GL), [error](GL), and/or [complete](GL) handler on it.
+- If no [error](GL) handler is provided, RxJS will assume all errors that are pushed are "unhandled" errors, and rethrow them on a different call stack.
+
+## Overview
+
+Chances are, the first time you encounter an [Observable](API), it is because you have had one returned to you by some other API. In this case, the most helpful thing for you to know is how to get values from the observable, and the semantics of [subscription](GL)
+
+> The first thing to know is, an observable doesn't "do" anything.
+
+An observable, in fact, is _not_ a "stream". An observable is a template for [subscription](GL), through which your code (the [consumer](GL)), will tell a [producer](GL) to start pushing values to it by subscribing to an observable, creating a [subscription](GL).
```

Comment:
Consider merging this definition with the definition of Observable from Chapter 4 to prevent later confusion. Maybe adding an additional sentence that says: "Once subscribed, you can think of an Observable as a container for a collection of values over time."